ArcadiaTeamsBot/Startup.cs

```diff
@@ -36,6 +40,7 @@ public void ConfigureServices(IServiceCollection services)
             services.AddTransient<IBot, Bot>();
             services.AddSingleton<IBotFrameworkHttpAdapter, BotAdapterWithErrorHandling>();
             services.AddSingleton(this.serviceDeskConfiguration);
+            services.AddSingleton(this.serviceDeskMappingConfiguration);

```

Comment:
Not the best way to register in DI container such common type as Dictionary. It will work, but violates some SOLID rules.

Reply:
Made it separate model with nested Dictionary.